{
  "modified": "0001-01-01T00:00:00Z",
  "published": "2016-10-30T00:00:00Z",
  "schema_version": "1.5.0",
  "id": "PSF-2016-10",
  "summary": "gettext.c2py()",
  "details": "Arbitrary code execution in ``gettext.c2py()``.\n",
  "affected": [
    {
      "ranges": [
        {
          "type": "GIT",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "07bcf05fcf3fd1d4001e8e3489162e6d67638285"
            },
            {
              "fixed": "a8760275bd59fb8d8be1f1bf05313fed31c08321"
            }
          ],
          "repo": "https://github.com/python/cpython"
        }
      ]
    }
  ],
  "references": [
    {
      "type": "REPORT",
      "url": "https://bugs.python.org/issue28563"
    }
  ],
  "database_specific": {
    "cwe_ids": []
  }
}
